charts/axosyslog/templates/service.yaml

```diff
@@ -1,19 +1,46 @@
 {{- if .Values.service.create }}
----
 apiVersion: v1
 kind: Service
 metadata:
-  name: "{{ template "axosyslog.fullname" . }}"
+  name: {{ template "axosyslog.fullname" . }}-syslog
 spec:
   selector:
-    app: "{{ template "axosyslog.fullname" . }}"
-    release: {{ .Release.Name | quote }}
+    app: {{ template "axosyslog.fullname" . }}-syslog
+  type: NodePort
```

Comment:
this could be configurable. ClusterIP (the default) and LoadBalancer make sense too

Reply:
can you point me to a sample how this is usually configured in other charts?